recipes/breakfest/arabic_omelette.md

```diff
@@ -0,0 +1,35 @@
+# Arabic Omellete
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-10%25-brightgreen.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Very%20Positive-brightgreen.svg)
```

Comment:
I wouldn't say the reception I witnessed was _very_ positive. Positive maybe.

I should really get my review app in order so we can get some anonymous feedback.

Reply:
How about we get the review app going and then we could know.
ask @m4v15 he could be your first  reviewer

Reply:
I think maybe @yvonne-liu and @Suuuuha had some of your omelette a few days ago.